### ISSUE TITLE
A10-C - CDU Display - CDU_Lines (SYS / LASTE  / WIND ) 

### ISSUE THREAD

**Issue by landre-cerp** [DCS-Skunkworks/dcs-bios#120]
I noticed that there's a mismatch when you go to LASTE page. 
On the CDU , press SYS , Then LASTE ( upper right choice ) , then WIND lower right 
in my Case , the Temp +10 i can see on line0 in the plane is not in A10C/CDU_LINE0 , but on CDU_LINE4 just before WINDEDIT 

<img width="992" alt="image" src="https://user-images.githubusercontent.com/49897630/188282370-c675774e-4de4-44d6-975c-e901ac390506.png">

DCS bug ?

**Comment by Cerppo** [comment]
And the "LASTE <-  just above the P1/2 , is not present in the CDU_LINE7 ?

**Comment by charliefoxtwo** [comment]
Hey @landre-cerp, I know it's been a year but I have just had the time to start looking into this if this still affects you.

This one is a weird one. In the DCS luas, they have three pages - `WIND`, `WIND1`, and `WIND2`. The information we receive is that they're showing `WIND1`, yet in game it appears to display `WIND` (the same goes for `WINDEDIT`, `WINDEDIT1`, and `WINDEDIT2`).

Do you know if the `WIND` and/or `WINDEDIT` potentially each have multiple pages? I'm not clear what these other layouts are supposed to be for. I have a potential fix, but it would basically be to overwrite `WIND1` with `WIND` and this may open up other issues if there do happen to be multiple wind pages.

**Comment by charliefoxtwo** [comment]
I think we're talking about different things. I'm wondering if there are like entirely different pages with different layouts (though the same information) also called WIND or WNDEDIT that you might be aware of.

If there aren't, then I can push a fix. If there are, then my fix may break them.

**Comment by landre-cerp** [comment]
i did not find other pages.

**Comment by charliefoxtwo** [comment]
alright, I can go ahead with the fix then. It's really a hack since it _seems_ the information we're receiving from ED is different from what's actually on the screen. This is why it could break things if there's some other part we're missing. In my limited testing though, everything seemed to work

**Comment by talbotmcinnis** [comment]
I'm late to the party, and I suspect the fix proposed is correct, but I'll toss in my thoughts in case this can be tuned:
The definition of the LINE1 is "Current IFFCC CALCULATED WIND AND TEMPERATURE".  WNDEDIT has 2 layouts:  For Page 1, you should have the IFFCC calculated info, RSK1 should show "WIND" and the "columns" should have a header, followed by three "rows" for data entry.  Page 2 drops the IFFCC part and column headers in return for four "rows" for data entry".

Comparing WNDEDIT page 1 vs 2 is interesting, as the IFFCC temp is in a different place, and wrong both times :)
![IMG_1306](https://github.com/DCSFlightpanels/dcs-bios/assets/3245082/adea3116-13f2-4290-9846-07e7dc5ad210)
![IMG_1305](https://github.com/DCSFlightpanels/dcs-bios/assets/3245082/6f9fda13-ff92-4e9a-8a1e-68ebd399f0b3)

**Comment by charliefoxtwo** [comment]
@talbotmcinnis feel free to test out the fix and let me know if it's correct!

**Comment by landre-cerp** [comment]
Hi charlie, i tested  and It's worse with the fix. 
it fixed the OAT on the Wind Page , where it  was not displayed correctly
Now on L2 at the right, it's Correct. 

But, it broke the display when you enter Altitudes using the 3 boxes on the left. 
on the 1st ligne The number below "ALT is not displayed" , on the 2nd line nothing is displayed
Windedit page is "empty"

**Comment by charliefoxtwo** [comment]
Hm. That doesn't sound right. I seem to recall the wind edit page displayed just fine in my testing.

Are you testing in the A10C or A10CII?

**Comment by landre-cerp** [comment]
A10c 2

**Comment by charliefoxtwo** [comment]
Unfortunately I don't know enough about the A-10 to be able to test and reproduce the issue you're referring to.

> But, it broke the display when you enter Altitudes using the 3 boxes on the left.

Could you give me exact instructions as to what page I should be on and exactly what I should press in order to do this? Then I can try to reproduce the issue and see what's going on. If you have a mission I could load up that already has the data entered, that would be even better.

> Windedit page is "empty"

I just double-checked the windedit page and the only thing I see missing is the text for the LSK 3R button - but maybe I need more data loaded in order to see what you're seeing.

**Comment by landre-cerp** [comment]
if that helps, the procedure is here 
https://youtu.be/r5hCfUZZJhc?si=vJf8e2f_WSlDQ7hu&t=351 
When he enters the 00 using the LSK,  etc..

**Comment by charliefoxtwo** [comment]
Thanks for providing that information.

I did some more digging and the solution I initially attempted was definitely not the right one. However, I'm not sure this can be _fully_ solved.

I've made some more changes in this pr (https://github.com/DCS-Skunkworks/dcs-bios/pull/236), including rolling back the previous changes. It also has screenshots to demonstrate how _most_ everything works, but at least one issue remains. Give that a look and let me know if that's at least an improvement.

It seems like we're dealing with an ED bug here where they're storing data for different items in the same field and during that time there's nothing we can do to get the outside air temperature.

Let me know what your thoughts are on these changes. I don't love that it's not completely fixed, but I'm not sure it's possible for us to completely fix it either.
